### DISCUSSION TITLE
How to make the HashtagNode linkable?

### DISCUSSION THREAD

**Discussion by novito** [facebook/lexical#5356] [Q&A]
Hello,

I am trying to use the `LexicalHashtagPlugin` and I am having a hard time navigating the code to understand how could I modify the `HashtagNode` so that it can be a link, so that when you click on the hashtag you are redirected to another page.

What would be a proper approach to do this? I am thinking of modifying the node to have a LinkNode wrapping it around it, but curious to know what your thoughts are.

**Comment by ivailop7** [comment]
Look into how the ClickableLinkPlugin is done in the repo.
